src/Parser/EnhancementShaman/Modules/ShamanCore/AlphaWolf.js

```diff
@@ -0,0 +1,41 @@
+import SPELLS from 'common/SPELLS';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import Module from 'Parser/Core/Module';
+
+class AlphaWolf extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  }
+
+  feralSpiritCastCount = 0;
+  feralSpiritCastFlag = false;
```

Comment:
These two could be prefixed with `_` to indicate private variables (since the main variable for this module is `missedAlphaWolfCount`) but it's just a style issue.

Reply:
"Flag" in the name can be omitted. If you want to make it clearer what it is how about `_didCastFeralSpirit`?